binaries/daemon/main.go

```diff
@@ -32,13 +34,16 @@ func main() {
 	if err != nil {
 		log.Fatal("error creating temp dir: ", err)
 	}
+	defer os.RemoveAll(tempDir.Dir)
```

Comment:
This'll make it hard to debug; I guess I was trusting that temp gets reaped... WDYT?